tests/queries/0_stateless/02711_soundex_function.sql

```diff
@@ -0,0 +1,13 @@
+select soundex('');
+select soundex('12345');
```

Comment:
It's slightly more convenient for future readers if the expected results are part of this file already, e.g.

```sql
select soundex('') = FixedString('0000');
```
(didn't run the example locally)

The expected result file will then contain a list of 1s.